test/functional/http/get/peers.js

```diff
@@ -69,6 +69,11 @@ describe('GET /peers', () => {
 			invalid: ['9999.999.999a'],
 			checkResponse: true,
 		},
+		protocolVersion: {
+			valid: ['999.999'],
+			invalid: ['9999.9999', '-1.-1', 'a.b'],
```

Comment:
What about null, undefined or empty string?

Reply:
So as far as I've seen this  validation is also performed on handshake time for inbound connections to check for valid WS headers, not only on the HTTP API. So we should allow null, undefined or empty string on `protocolVersion` to keep compatibility with older nodes. Therefore null, undefined or empty string values shouldn't be invalid. If one of those values is received on an updated node, it will be aware of it and therefore will continue to check for compatibility with the `version`.